fuzzing/fuzzer.go

```diff
@@ -324,6 +325,35 @@ func (f *Fuzzer) createTestChain() (*chain.TestChain, error) {
 		Balance: initBalance,
 	}
 
+	// Identify which contracts need to be predeployed to a deterministic address by iterating across the mapping
+	contractAddressOverrides := make(map[common.Hash]common.Address, 0)
+	for contractName, addrStr := range f.config.Fuzzing.PredeployedContracts {
+		found := false
+		// Try to find the associated compilation artifact
+		for _, contract := range f.contractDefinitions {
+			if contract.Name() == contractName {
+				// Hash the init bytecode (so that it can be easily identified in the EVM) and map it to the
```

Comment:
Long term, we should probably support duplicate contract names by allowing to match on a key like `File.sol:Contract` and warn if only the contract name is provided in the presence of duplicates

Reply:
Yes, agreed. Looping this into:
https://github.com/crytic/medusa/issues/39